cn.lpkt.serverbox.app.yaml

```diff
@@ -0,0 +1,39 @@
+app-id: cn.lpkt.serverbox.app
```

Comment:
![image](https://github.com/user-attachments/assets/3c7289c5-aab8-4945-bb52-e130cc08b779)


https://docs.flathub.org/docs/for-app-authors/requirements#application-id

Reply:
Also I can't connect to the website

![image](https://github.com/user-attachments/assets/50e2b4ef-8ff9-46f3-bcbe-6bd37779f280)